### PR TITLE
DDLS newline serialization in 750 and lower

### PR DESCRIPTION
**Pull request by jrodriguez-rc**
Fix for #1879

### CODE REVIEW

**Comment by larshp** [comment]
thanks, looks good to me 👍

**Comment by larshp** [comment]
candidate to be merged at same time as https://github.com/abapGit/abapGit/pull/4010

File: src/objects/zcl_abapgit_object_ddls.clas.abap [hunk 1]
```diff
@@ -428,4 +433,33 @@ CLASS zcl_abapgit_object_ddls IMPLEMENTATION.
...
+
+    lv_len = strlen( cv_string ) - 2.
+    lv_lastchar2 = cv_string+lv_len(1).
+
+    " only add a line break, if the last character is unequal to cr_lf and newline !
+    IF lv_lastchar1 <> cl_abap_char_utilities=>cr_lf AND lv_lastchar1 <> cl_abap_char_utilities=>newline AND
+        lv_lastchar1 <> space OR
+        ( lv_lastchar1 = space AND
+          ( lv_lastchar2 <> cl_abap_char_utilities=>cr_lf AND lv_lastchar2 <> cl_abap_char_utilities=>newline ) ).
+      cv_string = |{ cv_string }{ cl_abap_char_utilities=>cr_lf }|.
```
[the remark this answers is not in the document]
Comment (jrodriguez-rc) [reply src/objects/zcl_abapgit_object_ddls.clas.abap:477]: The standard uses cr_lf
image

[the remark this answers is not in the document]
Comment (mbtools) [reply src/objects/zcl_abapgit_object_ddls.clas.abap:477]: Thx

**Review by mbtools — APPROVED** [review]
👍

**Pull request merged**